go/roothash/api/commitment/pool.go

```diff
@@ -483,6 +514,9 @@ func (p *Pool) ResolveDiscrepancy() (OpenCommitment, error) {
 	}
 
 	minVotes := (backupNodes / 2) + 1
+	if failuresTally >= minVotes {
+		return nil, ErrInsufficientVotes // TODO: new error?
```

Comment:
Yeah it would be good to have a new error code, not sure how it should be called.

Reply:
Used `ErrMajorityFailure` for now